themes/cpssd/static/css/main.css

```diff
@@ -0,0 +1,58 @@
+body {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+}
+
+header h1 {
+  font-size: 3rem;
+}
+
+header::after {
+  display: block;
+  content: '';
+  width: 15%;
+  height: 1px;
+  margin: 1rem auto;
+  background-color: lightgrey;
```

Comment:
does `lightgrey` really work instead of `lightgray`?

Reply:
https://www.w3schools.com/colors/color_tryit.asp?color=LightGrey

Reply:
[CSS took colour names from X11, including duplicates](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value). So yeah, it works, TIL.